### ISSUE TITLE
/usr/local/bin/node: bad option: --inspect

### ISSUE THREAD

**Issue by PixelT** [willyelm/xatom-debug#1]
After click run, I have error in console:
```
/usr/local/bin/node: bad option: --inspect
Error: connection ECONNREFUSED 127.0.0.1:5858
```
?

**Comment by willyelm** [comment]
which node version are you using?

**Comment by PixelT** [comment]
node v4.2.1

**Comment by willyelm** [comment]
yes --inspect is only available on node 6.3+, If you still need to stay with the 4.2 version please use `atom-bugs-legacy`
